component/admin/libraries/jevbuttons.php

```diff
@@ -13,8 +13,9 @@
 // Check to ensure this file is within the rest of the framework
 defined('JPATH_BASE') or die();
 jimport('joomla.html.toolbar.button');
```

Comment:
WE need to make sure that cms.toolbar.button is loaded rather than joomla.html.toolbar.button or does the auto-loader take care of this?